apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js

```diff
@@ -2,8 +2,8 @@
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
```

Comment:
__nit:__ it would be great if you can update mozMobileMessage.retrieveMMS in desktop mock so that it returns Promise....